sdk/monitor/monitor-opentelemetry/package.json

```diff
@@ -81,7 +81,8 @@
     "rimraf": "^5.0.5",
     "sinon": "^17.0.0",
     "tsx": "^4.7.1",
-    "typescript": "~5.5.3"
+    "typescript": "~5.5.3",
+    "@opentelemetry/instrumentation-fs": "^0.14.0"
```

Comment:
Please remove this

Reply:
I'm using this dev dependency to unit test OTel instrumentations not included in the distro. I can try spoofing the list of registered instrumentations, but I believe this allows us to do the most complete unit test of this feature.